### DISCUSSION TITLE
Proxy fails because nginx (coolify container) uses port 80

### DISCUSSION THREAD

**Discussion by Arctomachine** [coollabsio/coolify#3107] [Q&A]
Port 80 is in use.
Not on host, but inside coolify container, nginx uses it.
How should I use proxy then? Is it intended?
![image](https://github.com/user-attachments/assets/bc8e52fa-9403-4120-a75b-c4b23c587217)

**Comment by Arctomachine** [comment]
False alarm probably. I fixed it somehow by changing hosts file on host system. Could be docker misconfigured previously

**Reply by nyeeldzn** [reply]
How did you fix this?
I'm with same problem...

**Reply by Arctomachine** [reply]
No idea actually. I made sure that docker records in hosts file, ip in docker settings and real local ip of computer match. Then it worked, I got proxy container automatically after next docker restart

**Reply by nyeeldzn** [reply]
i undestood... thank you very much anyway!
